src/metabase/query_processor/middleware/cache.clj

```diff
@@ -139,8 +139,7 @@
 
         ([acc row]
          (if (map? row)
-           (do (vreset! final-metadata row)
-               (rf acc))
```

Comment:
took quite a while to realize we were doing this here. We can just not process this row and save it in the final-metadata